web/modules/custom/demo_content/demo_content.install

```diff
@@ -19,6 +20,15 @@ function demo_content_install() {
  * Creates demo content for the joinup project.
  */
 function demo_content_create_demo_content() {
+  // Create demo users.
+  foreach (demo_content_get_user_data() as $user_data) {
+    $facilitator = user_load_by_name($user_data['name']);
```

Comment:
Why would this user already exist?

This module should only be enabled on clean installations, and only on the UAT environment. If there is already data present (e.g. users) then this will cause data loss.

I'd rather have this fail with a fatal error than silently overwrite existing users on who knows which environment somebody might enable this module on.

I think it would be safer to remove this entire module from the repo and move it to a dev-dependency so it will not be present on production environments, but that is out of scope for this issue.